README.id.md

```diff
@@ -0,0 +1,24 @@
+[![Github Stars](https://img.shields.io/badge/stars-nominate-brightgreen?logo=github)](https://stars.github.com/nominate/)
+
+<h1 align="center">Pusat sumber informasi mengenai ilmu Kecerdasan Buatan (AI)</h1>
+
+<div align="center">
+  
+[English](README.md) | [Hindi](README.hi.md) | [Italiano](README.it.md) | [French](README.fr.md) | [Indonesian](README.id.md)
+  
+</div>
+
+## 👋 Halo
+
+Repositori ini berisikan mengenai kumpulan informasi mengenai Kecerdasan Buatan (AI) dari berbagai sumber. Informasi yang disampaikan disini ditujukan untuk kaum pemula maupun yang sudah profesional dibidang ilmu Kecerdasan Buatan. 
+
+## 🚧 Bantu kami dalam berkontribusi
+
+Bahasa Inggris bukanlah satu-satunya bahasa di dunia. Ada banyak orang yang tidak bisa berbahasa inggris. Banyak pembuat konten tidak membuat konten mereka dalam bahasa Inggris. Oleh karena itu, kami ingin membuat kumpulan informasi yang terpisah yang dapat membantu orang yang ingin mempelajari Machine Learning (ML) dan tidak memiliki kemampuan untuk mengartikan bahasa dari konten tersebut. Bahasa Indonesia adalah bahasa selanjutnya yang ingin kami dukung. Ayo bantu kami dalam [membangun](https://github.com/SkalskiP/courses/discussions/22) kumpulan sumber informasi dalam Bahasa Indonesia. 
```

Comment:
Could we update the discussion link from https://github.com/SkalskiP/courses/discussions/22 to https://github.com/SkalskiP/courses/discussions/34? The discussion you linked is about Hindi resources. I just created a separate one for Indonesian.

Reply:
Hi. 
ah, thanks for noticing that one. I Updated the PR.